backend/src/main/scala/cromwell/backend/validation/RuntimeAttributesValidation.scala

```diff
@@ -78,46 +75,62 @@ object RuntimeAttributesValidation {
   def withDefault[ValidatedType](validation: RuntimeAttributesValidation[ValidatedType],
                                  default: WdlValue): RuntimeAttributesValidation[ValidatedType] = {
     new RuntimeAttributesValidation[ValidatedType] {
-      override def key = validation.key
+      override def key: String = validation.key
+
+      override def coercion: Traversable[WdlType] = validation.coercion
 
-      override def coercion = validation.coercion
+      override protected def validateValue: PartialFunction[WdlValue, ErrorOr[ValidatedType]] =
+        validation.validateValuePackagePrivate
 
-      override protected def validateValue = validation.validateValuePackagePrivate
+      override protected def validateExpression: PartialFunction[WdlValue, Boolean] =
+        validation.validateExpressionPackagePrivate
 
-      override protected def validateExpression = validation.validateExpressionPackagePrivate
+      override protected def invalidValueMessage(value: WdlValue): String =
+        validation.invalidValueMessagePackagePrivate(value)
 
-      override def staticDefaultOption = Option(default)
+      override protected def missingValueMessage: String = validation.missingValueMessage
 
-      override protected def failureMessage = validation.failureMessagePackagePrivate
+      override protected def usedInCallCaching: Boolean = validation.usedInCallCachingPackagePrivate
+
+      override protected def staticDefaultOption = Option(default)
     }
   }
 
   def optional[ValidatedType](validation: RuntimeAttributesValidation[ValidatedType]):
   OptionalRuntimeAttributesValidation[ValidatedType] = {
     new OptionalRuntimeAttributesValidation[ValidatedType] {
-      override def key = validation.key
+      override def key: String = validation.key
+
+      override def coercion: Traversable[WdlType] = validation.coercion
 
-      override def coercion = validation.coercion
+      override protected def validateOption: PartialFunction[WdlValue, ErrorOr[ValidatedType]] =
+        validation.validateValuePackagePrivate
```

Comment:
ToL not sure I heart the pervasive package private aliasing pattern (seems kind of complicated), but I realize that's preexisting.